ui/src/audioplayer/Player.js

```diff
@@ -31,6 +31,11 @@ const useStyle = makeStyles((theme) => ({
   },
   player: {
     display: (props) => (props.visible ? 'block' : 'none'),
+    '@media screen and (max-width:810px)': {
+      '& .progress-bar-content': {
+        display: 'none!important',
```

Comment:
Can it be done without the `!important`?

Reply:
Sure, it could be done without `!important`. I will remove it in the next commit.